src/main/webapp/WEB-INF/category.jsp

```diff
@@ -28,6 +28,11 @@
       #map {
         height: 75px;
       }
+		
+	  .message_map {
+		height: 75px;
```

Comment:
How about just putting this in user-page.css with the other message related styles instead of here and below so you only need to specify it in one place? Sorry I didn't notice you were defining this in 2 places before..